fontconfig.yaml

```diff
@@ -10,6 +10,14 @@ package:
         - "*"
       attestation: TODO
       license: MIT
+  scriptlets:
+    trigger:
```

Comment:
@kaniini is this correct?  I'd missed these triggers before.  I've seen some examples already merged in wolfi but not 100% how triggers work to test this.